src/parcsr_ls/par_ilu.c

```diff
@@ -81,6 +81,9 @@ hypre_ILUCreate()
    hypre_ParILUDataNumIterations(ilu_data) = 0;
 
    hypre_ParILUDataMaxIter(ilu_data) = 20;
+   hypre_ParILUDataTriSolve(ilu_data) = 0;
+   hypre_ParILUDataLowerJacobiIters(ilu_data) = 5;
```

Comment:
Is 5 iterations a good number here?

Reply:
It depends on the matrix.  Basically the rate at which elements decay from the diagonal determines how many iterations are needed.